### DISCUSSION TITLE
[Discussion] How to implement inpainting?

### DISCUSSION THREAD

**Discussion by lllyasviel** [Mikubill/sd-webui-controlnet#788] [General]
I think all other models are good now.
Does our current code architecture support a good way to do inpainting?
Or it is impossible with A1111 plugin design?

**Comment by ljleb** [comment]
Are you referring to inpainting models or the img2img inpaint tab?

For inpaint models, we get a tensor size mismatch when trying to use them. Maybe we can look at the shape difference to detect them?
